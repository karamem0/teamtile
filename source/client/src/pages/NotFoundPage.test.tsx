//
// Copyright (c) 2023 karamem0
//
// This software is released under the MIT License.
//
// https://github.com/karamem0/teamtile/blob/main/LICENSE
//

import React from 'react';

import { render, screen } from '@testing-library/react';

import NotFoundPage from './NotFoundPage.presenter';

test('create shapshot', async () => {
  const params = {};
  render(<NotFoundPage {...params} />);
  expect((await screen.findAllByText(/^.*$/))[0]).toMatchSnapshot();
});
