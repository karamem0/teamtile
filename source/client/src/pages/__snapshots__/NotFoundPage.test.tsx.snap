// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`create shapshot 1`] = `
.emotion-0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-flex-flow: column;
  -webkit-flex-flow: column;
  -ms-flex-flow: column;
  flex-flow: column;
}

.emotion-1 {
  font-size: 3rem;
  line-height: 3rem;
  text-align: center;
}

<body>
  <div>
    <div
      class="emotion-0"
    >
      <h1
        class="ui-header emotion-1"
        data-uses-unhanded-props="true"
      >
        <span
          dir="auto"
        >
          404
        </span>
        <p
          class="ui-header__description"
          data-uses-unhanded-props="true"
          dir="auto"
        >
          page not found
        </p>
      </h1>
    </div>
  </div>
</body>
`;
